docs/docusaurus/credentials/present-proof.md

```diff
@@ -121,6 +121,7 @@ curl -X 'GET' 'http://localhost:8090/prism-agent/present-proof/presentations' \
 
 The Holder/Prover can then accept a specific request, generate the proof, and send it to the Verifier PRISM Agent by making a `PATCH` request to the [`/present-proof/presentations/{id}`](/agent-api/#tag/Present-Proof/operation/updatePresentation) endpoint:
 
+#### JWT
```

Comment:
For the sake of consistency, I propose to use [Docusaurus Tabs](https://docusaurus.io/docs/markdown-features/tabs) as in the issue flow documentation [here](https://github.com/hyperledger-labs/open-enterprise-agent/blob/19c42b10188d1e0242bdceb1f89b6410dcc05353/docs/docusaurus/credentials/issue.md?plain=1#L22).

Reply:
fixed